code/modules/admin/tabs/event_tab.dm

```diff
@@ -145,87 +145,36 @@
 	if(!check_rights(R_EVENT))
 		return
 
-	var/tag = tgui_input_list(usr, "Which ERT shuttle should be force launched?", "Select an ERT Shuttle:", list("Distress", "Distress PMC", "Distress UPP", "Distress_Big", "Distress_Small"))
+	var/tag = tgui_input_list(usr, "Which ERT shuttle should be force launched?", "Select an ERT Shuttle:", list("Response Shuttle", "PMC Shuttle", "UPP Shuttle", "Boarding Shuttle", "Mini Shuttle"))
 	if(!tag) return
 
-	var/newShuttle = FALSE
-	if(tag in list("Distress", "Distress PMC", "Distress UPP"))
-		newShuttle = TRUE
 	var/list/shuttle_map = list(
-		"Distress" = MOBILE_SHUTTLE_ID_ERT1,
-		"Distress PMC" = MOBILE_SHUTTLE_ID_ERT2,
-		"Distress UPP" = MOBILE_SHUTTLE_ID_ERT3,
+		"Response Shuttle" = MOBILE_SHUTTLE_ID_ERT1,
+		"PMC Shuttle" = MOBILE_SHUTTLE_ID_ERT2,
+		"UPP Shuttle" = MOBILE_SHUTTLE_ID_ERT3,
+		"Mini Shuttle" = MOBILE_SHUTTLE_ID_ERT_SMALL,
+		"Boarding Shuttle" = MOBILE_SHUTTLE_ID_ERT_BIG
 	)
```

Comment:
can remove this if the other suggestion is applied

Reply:
Used previous solution, nice catch!